totest-manager.py

```diff
@@ -399,13 +399,19 @@ def package_ok(self, project, package, repository, arch):
         url = self.api.makeurl(['build', project, '_result'], query)
         f = self.api.retried_GET(url)
         root = ET.parse(f).getroot()
+        succeeded = False
```

Comment:
Perhaps just utilize xpath (or in this case even just `ET.find()`) instead of the loop and variable. Something like the following:

https://github.com/openSUSE/openSUSE-release-tools/blob/0d9cde5a9089f11e9747ffecfb5b1c034a248388/osclib/core.py#L429-L431

```suggestion
        succeeded = len(root.findall('result')) and not len(root.findall('result[@status!="succeeded"]))
```